gateway/src/main/webapp/app/core/auth/account.service.ts

```diff
@@ -145,10 +145,7 @@ export class AccountService {
   }
 
   isLoggedAs(): boolean {
-    if (this.isIdentityResolved() && this.userIdentity && this.userIdentity.loggedAs) {
-      return true;
-    }
-    return false;
+    return !!(this.isIdentityResolved() && this.userIdentity && this.userIdentity.loggedAs);
```

Comment:
@DanielPalafox I think this is not right, do we really need the double negation? `!!`